app/src/main/java/org/schabi/newpipe/local/subscription/SubscriptionManager.kt

```diff
@@ -66,6 +69,16 @@ class SubscriptionManager(context: Context) {
             }
         }
 
+    fun updateNotificationMode(serviceId: Int, url: String?, @NotificationMode mode: Int): Completable {
+        return subscriptionTable().getSubscription(serviceId, url!!)
```

Comment:
@TobiGr 
Do you know why the ``url``-parameter is nullable? 
Or more specifically: Is there any reason not to remove the ``?`` from the ``url``-parameter?

Reply:
I have no idea. I don't see any reason to not remove the `?` and `!!`. Go ahead.